### PR TITLE
GMP Documentation: Apache HTTP

### PR DESCRIPTION
**Pull request by algchoo**
This PR includes the following for the Apache HTTP GMP integration:
* documentation

### CODE REVIEW

File: integrations/apache/documentation.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    - port: prometheus
+      scheme: http
+      interval: 30s
+      path: /metrics
+    selector:
+      matchLabels:
+        app.kubernetes.io/name: httpd
+additional_prereq_info: |
+  Ensure that the values of the `port` and `matchLabels` fields match those of the {{app_name_short}} pods you want to monitor.
+alerts_config: {% includecode content_path="stackdriver/docs/managed-prometheus/exporters/configs/httpd_alerts.yaml" %}
```
Comment (varun-c) [inline integrations/apache/documentation.yaml:89]: IIUC, this field refers to a separate yaml file that contains the alerts. Did you mean to add that file as part of this PR or will that be a separate PR? If it is the latter, we can remove this for now.

Comment (algchoo) [reply integrations/apache/documentation.yaml]: The case here was that the existing alerts were not easily ported over using the available prometheus metrics. I've removed both sections. Thank you for the feedback, I appreciate it!

**Review by varun-c — APPROVED** [review]
